container/nginx/njs/badbits.js

```diff
@@ -0,0 +1,34 @@
+const fs = require("fs");
+const crypto = require("crypto");
```

Comment:
```suggestion
import fs from "node:fs";
import crypto from "node:crypto";
```

I'm not sure why it worked before, but one shouldn't mix commonjs and esm module syntax

Reply:
njs isn't exactly node.js, but its own thins, so scoping by `node:*` won't work